src/structures/Widget.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const Base = require('./Base');
+const Collection = require('../util/Collection');
+
+/**
+ * Represents a Widget.
+ */
+class Widget extends Base {
+  /**
+   * @param {Client} client - The instantiating client
+   * @param {Object} data - The raw data
+   */
+  constructor(client, data) {
+    super(client);
+    this._patch(data);
+  }
+  /**
+   * Builds the widget with the provided data.
+   * @param {*} data The raw data of the widget
+   * @private
+   */
+  _patch(data) {
+    /**
+     * The id of the guild.
+     * @type {string}
```

Comment:
shouldn't all the ids be typed as "Snowflake" instead of "string"?

Reply:
Done in d3e90ba